cryptarchia/cryptarchia.py

```diff
@@ -156,15 +161,9 @@ class BlockHeader:
     content_size: int
     content_id: Id
     leader_proof: MockLeaderProof
+    orphaned_proofs: List[MockLeaderProof] = field(default_factory=list)
```

Comment:
This is a list of block headers no?
```suggestion
    orphaned_proofs: List[BlockHeader] = field(default_factory=list)
```